examples/gnrc_minimal/main.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2015 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Showing minimum memory footprint of gnrc network stack
+ *
+ * @author      Oliver Hahm <oliver.hahm@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "msg.h"
+#include "net/ipv6/addr.h"
+#include "net/gnrc/netif.h"
+#include "net/gnrc/ipv6/netif.h"
+
+int main(void)
+{
+    char ipv6_addr[IPV6_ADDR_MAX_STR_LEN];
+    kernel_pid_t ifs[GNRC_NETIF_NUMOF];
+
+    puts("RIOT network stack example application");
+
+    /* get the first IPv6 interface and prints its address */
+    size_t numof = gnrc_netif_get(ifs);
+    if (numof > 0) {
+        gnrc_ipv6_netif_t *entry = gnrc_ipv6_netif_get(ifs[0]);
+        for (int i = 0; i < GNRC_IPV6_NETIF_ADDR_NUMOF; i++) {
```

Comment:
you may also want to reduce `GNRC_IPV6_NETIF_ADDR_NUMOF` to something around `4` for
```
ff02::1/128
ff02::1:XXX/64
ff02::2/128
fe80::XXX/64
```

I think the default value is around 6-7 and each `gnrc_ipv6_netif_addr_t` struct is in the range of ~54 bytes